docs/reference/tab-widgets/troubleshooting/data/add-tier.asciidoc

```diff
@@ -0,0 +1,127 @@
+//////////////////////////
+
+[source,console]
+--------------------------------------------------
+PUT my-index-000001
+
+--------------------------------------------------
+// TESTSETUP
+
+[source,console]
+--------------------------------------------------
+DELETE my-index-000001
+--------------------------------------------------
+// TEARDOWN
+
+//////////////////////////
+
+
+// tag::cloud[]
+In order to get the shards assigned we need enable a new tier in the deployment.
```

Comment:
I don't think we're "enabling" a tier, so much as adding to it?

```suggestion
In order to get the shards assigned we need add nodes for a specific tier in the deployment.
```

Reply:
I think we are enabling as it's missing completely (and in cloud we say "add capacity" to a tier) - the concept of `nodes` is missing in cloud.